### PR TITLE
Add more task regressions

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Adds *many* more task regressions.

**Testing steps**
CI

### CODE REVIEW

File: parlai/tasks/blended_skill_talk/agents.py [hunk 1]
```diff
@@ -122,12 +122,14 @@ class ConvAI2PersonaTopicifierTeacher(Convai2DefaultTeacher):
...
+            warn_once(
+                'Warning: the BST Convai2 teacher is not compatible with '
+                'streaming datatypes. Switching to nonstreaming.'
+            )
             # StreamDialogData works by reading directly from a text file without any
             # alteration, but this teacher must append a WoW topic string to the context
             # of the first example of each episode.
-            assert opt['datatype'].endswith(':stream')
-            opt['datatype'] = opt['datatype'][: -len(':stream')]
+            opt['datatype'] = opt['datatype'].replace(':stream', '')
```
Comment (klshuster) [inline parlai/tasks/blended_skill_talk/agents.py:132]: thanks for this, feels more user friendly

**Review by klshuster — APPROVED** [review]
🧪

File: conftest.py [hunk 2]
```diff
@@ -35,7 +36,12 @@ def get_class_name(item):
             module_name = parent.module.__name__
             break
 
-    if class_name:
+    # heuristic:
+    # - better to group gpu and task tests, since tests from those modules
```
Comment (EricMichaelSmith) [inline conftest.py:40]: 👍

**Review by EricMichaelSmith — APPROVED** [review]
Great to have

**Pull request merged**